.eslintrc

```diff
@@ -76,7 +76,7 @@
     "require-await": "error",
     "rest-spread-spacing": ["error", "never"],
     "react/prop-types": "error",
-    "react-hooks/exhaustive-deps": ["error", {"additionalHooks": "useSelect"}],
+    "react-hooks/exhaustive-deps": ["error", {"additionalHooks": "useSelect|useBatchingCallback"}],
```

Comment:
While you're at it, could you actually remove the `useSelect` bit? It's a leftover from the Gutenberg era.

Reply:
done

Reply:
done

Reply:
Thanks!